Source/Resources/en.lproj/Localizable.strings

```diff
@@ -10,18 +10,18 @@
 "billingAddressTitle" = "Billing address";
 "BillingFormCountryButtonTitle" = "Please select a country";
 "BillingFormOptional" = "Optional";
-"PaymentFormSummaryViewHintText" = "We need this information as an additional security measure to verify this card.";
+"PaymentHeaderSubtitle" = "We need this information as an additional security measure to verify this card.";
+"PaymentHeaderTitle" = "Payment details";
+"PaymentFormSummaryViewHintText" = "Visa, Mastercard and American Express accepted.";
```

Comment:
IMHO, Localisation doesn't need to be described because they are meaningful statements or words. 
In case of auto generated UI elements like 
```
"T2l-21-qwk.normalTitle" = "Go to payment page"; 
```
then we add some information for `T2l-21-qwk.normalTitle`

Reply:
This might make sense when you have source code to use, but try receiving a localisation request as a translator. There's no source code or anything. And in different languages the same word may be expressed in many ways, so any detail you can offer to describe purpose and use type will reduce likelihood of awkward localisations.

For reference [Apple docs](https://developer.apple.com/localization/#localize) strongly and repeatedly mention things like 
> Providing the service (localisation service) with context about your app and content can help increase efficiency and minimize translation errors.

This is only 1 page example, but I would be interested to find any Apple demo where they don't encourage and show comments for localisations 🤔.

Reply:
Actually i agree when we are supporting other languages like Arabic or German.

Reply:
Does that mean you plan to:
- start now
- bring it up for conversation with iOS devs
- wait for it to be needed

Translations will be done with what we have, so the latter we start, the more out of context strings there will be and higher likelihood of rubbish translations being sent to customers.

Reply:
This is a cool idea. 

Let's review it as a Spike separately as part of Frames and resolve separately. Keep the localisations as-is for now and we can explore adding these sorts of descriptions to all localisations as part of another ticket.